addons/loki/definitions/collect-logs.cue

```diff
@@ -3,7 +3,7 @@ import (
 	"path"
 )
 
-"collect-logs": {
+"log-collect-file": {
```

Comment:
If change this name, also change the filename. BTW, `log-collect-file` is weird, suggest using `collect-file-logs` or `file-logs-collector`.

Reply:
good suggestion.

Reply:
good suggestion.